tests/providers/amazon/aws/executors/batch/test_batch_executor.py

```diff
@@ -626,7 +626,7 @@ def _unset_conf():
     def teardown_method(self) -> None:
         self._unset_conf()
 
-    def test_validate_config_defaults(self):
+    def test_validate_config_defaults(self, require_running_tests_from_sources):
```

Comment:
@o-nikolas  -> for some reason, when tests are run using airlfow installed from package, those config defaults are not working properly. It would need to be revised and checked why. The way how to check it is to follow the instructions I added in `contributing-docs/testing/unit_tests.rst`  after removing the `require_running_tests_from_sources` fixture.

Reply:
Hmm  @o-nikolas  I think the recent changes **broke** the current tests with executor simply. This might mean for example that current executor implementation (in main) will not work with Airflow 2.9 - might be accidental change, but with this one, our test suite works essentiallly as "client" for diffferent airflow versions, so we should be able use the tests to detect such cases (And correct the test to do the right thing - as expected from those who write the executors). Would love if you could take a look at this one :)

Reply:
OK. I think this one really needs to remain as it is. This test really tests consistency between `provider.yaml'  configuration and actual configuration defaults, and those will not be possible when airflow is iinstalled from sources, because ... there is no provider.yaml there. So we can easily skip that tests when installing airflow from packages.